setup.py

```diff
@@ -9,7 +9,8 @@
 
 install_requires = [
     'xattr',
-    'psutil'
+    'psutil',
+    'scandir;python_version<"3.5"'
```

Comment:
@traylenator when releasing the new RPM version, we should remember to encode this. Scandir is already included  with py35, otherwise: `python2-scandir` is needed.

Reply:
How come a python36-scandir exists in epel7 , should this not actually exist?